src/template-pages/content-page.html

```diff
@@ -67,6 +67,45 @@ <h1>Queensland Government Web Template HTML Template</h1>
                     <p>[Put more content here]</p>
 
                     <h2>To use this template</h2>
+                    <div class="qg-location">
```

Comment:
All this needs to be deleted. Should not be part of page template

Reply:
I think this is fine because If we say page content then it should contain some content to display page layout styles on various breakpoints etc for testing.